### ISSUE TITLE
Support using the inventory when not running in a container

### ISSUE THREAD

**Issue by masayag** [openshift/assisted-installer-agent#249]
In the k4e-device-worker project, we rely on the inventory to gather HW info from the device, see:
https://github.com/jakub-dzon/k4e-device-worker/blob/main/internal/hardware/hardware.go#L13
However, since the device-worker runs as a process and not as a container, there are few failures to collect certain data, e.g.:
```
Oct 17 19:00:06 localhost.localdomain yggdrasild[1000]: [yggdrasild] 2021/10/17 19:00:06 [/usr/libexec/yggdrasil/device-worker] WARNING: Unable to read product_family: open /host/sys/class/dmi/id/product_family: no such file or directory
```

We'd like the option not to set the Chroot as `/host` when not running as a container.

**Comment by omertuc** [comment]
Resolved by #248

**Comment by masayag** [comment]
Fixed by https://github.com/openshift/assisted-installer-agent/pull/248
